MahApps.Metro/Controls/RangeSlider.cs

```diff
@@ -2025,17 +2025,15 @@ private Double CalculateNextTick(Direction dir, double chekingValue, double dist
             if (!IsMoveToPointEnabled)
             {
                 //Check if current value is exactly Tick value or it situated between Ticks
-                if (!IsDoubleCloseToInt((chekingValue - Minimum) / TickFrequency))
+                if (!IsDoubleCloseToInt((chekingValue + distance - Minimum) / TickFrequency))
```

Comment:
While we're in here making this change, I have a couple of suggestions:

 - can we extract `(chekingValue + distance - Minimum)` into a local variable that matches up with that documented comment, for better readability - perhaps a name like `currentValueChanged`?
 - `chekingValue` should be renamed to `checkingValue` - that might have some flow-on effects...